Automattic/wp-calypso
# Plans: <ExpandedNudge> component

This creates a reusable `ExpandedNudge` component that can be safely dropped in without much effort. Resolves #7563 and almost fulfills  #7648,  #7651, #7647

Component is smart and tries to infer as much as possible from redux.

No wunctional changes

# Testing

1. Go to http://calypso.localhost:3000/settings/seo/:site on a non-bbusiness site
2. Confirm there are no functional changes

CC @rodrigoi @lamosty @rralian @gwwar
